File: android/api-tests/src/test/java/com/revenuecat/apitests/java/EntitlementInfosMapperApiTests.java
package com.revenuecat.apitests.java.mappers;

import com.revenuecat.purchases.EntitlementInfos;
import com.revenuecat.purchases.hybridcommon.mappers.EntitlementInfosMapperKt;

import java.util.Map;

@SuppressWarnings("unused")
class EntitlementInfosMapperApiTests {
    private void checkMap(EntitlementInfos entitlementInfos) {
        Map<String, Object> map = EntitlementInfosMapperKt.map(entitlementInfos);
    }
}

File: android/api-tests/src/test/java/com/revenuecat/apitests/kotlin/mappers/EntitlementInfosMapperApiTests.kt
package com.revenuecat.apitests.kotlin.mappers

import com.revenuecat.purchases.EntitlementInfos
import com.revenuecat.purchases.hybridcommon.mappers.map

@Suppress("unused", "UNUSED_VARIABLE")
private class EntitlementInfosMapperApiTests {
    fun checkMap(entitlementInfos: EntitlementInfos) {
        val map: Map<String, Any?> = entitlementInfos.map()
    }
}
